Arrowgene.Ddon.Database/Sql/Core/Migration/00000011_BinaryDataMigration.cs

```diff
@@ -0,0 +1,48 @@
+using Arrowgene.Ddon.Shared.Entity;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
+using Arrowgene.Logging;
+using System.Data.Common;
+
+namespace Arrowgene.Ddon.Database.Sql.Core.Migration
+{
+    public class BinaryDataMigration : IMigrationStrategy
+    {
+        private static readonly ILogger Logger = LogProvider.Logger<Logger>(typeof(BinaryDataMigration));
+
+        public uint From => 10;
+        public uint To => 11;
+
+        private readonly DatabaseSetting DatabaseSetting;
+
+        public BinaryDataMigration(DatabaseSetting databaseSetting)
+        {
+            DatabaseSetting = databaseSetting;
+        }
+
+        public bool Migrate(IDatabase db, DbConnection conn)
+        {
+            string adaptedSchema = DdonDatabaseBuilder.GetAdaptedSchema(DatabaseSetting, "Script/binarydata_migration.sql");
+            db.Execute(conn, adaptedSchema);
+
+            byte[] blankBlob = new byte[0x400];
```

Comment:
Instead of using the hardcoded number you could use C2SBinarySaveSetCharacterBinSaveDataReq.ARRAY_SIZE to be more explicit about why that number was used